Dockerfile

```diff
@@ -25,20 +22,30 @@ RUN DEBIAN_FRONTEND=noninteractive && \
   libopenoffice-oodoc-perl \
   postgresql-client \
   ssmtp \
-  lsb-release
+  lsb-release \
+  && apt-get autoremove -y \
+  && apt-get autoclean \
+  && rm -rf /var/lib/apt/lists/*
 
 
 # Build time variables
 ENV LSMB_VERSION master
 ENV NODE_PATH /usr/local/lib/node_modules
-ENV DEBIAN_FRONTEND=noninteractive
 
 ARG CACHEBUST
 
 
+###########################################################
 # Java & Nodejs for doing Dojo build
 # Uglify needs to be installed right before 'make dojo'?!
-RUN apt-get -y install git make gcc libperl-dev npm curl && \
+
+# These packages are only needed during the dojo build
+ENV DOJO_Build_Deps git make gcc libperl-dev npm curl
+# These packages can be removed after the dojo build
+ENV DOJO_Build_Deps_removal ${DOJO_Build_Deps} nodejs
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND="noninteractive" apt-get -y install ${DOJO_Build_Deps} && \
```

Comment:
this debian frontend setting applies only to a single command, but all `apt-get` invocations need it.